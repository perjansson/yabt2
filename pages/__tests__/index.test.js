import { shallow } from 'enzyme'
import React from 'react'

import Index from '../index'

describe('Index', () => {
    it('should render with burger images', () => {
        const wrapper = shallow(<Index />)
        expect(wrapper).toMatchSnapshot()
    })
})
