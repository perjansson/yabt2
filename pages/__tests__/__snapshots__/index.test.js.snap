// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render with burger images 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(Index) />
</Provider>
`;
